.github/pull_request_template.md

```diff
@@ -15,9 +15,11 @@ Please reference any related feature requests, issues, or PRs here. For example,
 ## How is this patch tested?
 
 <!--
-If you're unsure about what to test, where to add tests, or how to run tests, please feel free to ask. We'd be happy to help.
+If you're unsure about what to test, where to add tests, or how to run tests, please feel free to ask.
 -->
-- [ ] I have written tests (not required for typo or doc fix) and confirmed the proposed feature/bug-fix/change works.
+- [ ] Existing unit/integration tests
+- [ ] New unit/integration tests
+- [ ] Manual tests (describe details below)
```

Comment:
```suggestion
- [ ] Manual tests (describe details below and ensure that results are attached via PR comments)
```

Reply:
I think it's fine if the results are expressed in the description rather than as separate comments.